### PR TITLE
Integrate the tickets API at add-order route

### LINKED ISSUE #735: Implement the tickets API on the add orders route.
**I'm submitting a ...**  (check one with "x")
- [ ] bug report
- [x] feature request
- [ ] support request => Please do not submit support requests here, instead ask your query in out Gitter channel at https://gitter.im/fossasia/open-event-frontend
addorder

### PR DESCRIPTION
**Pull request by sumedh123**
#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

Fixes #735

### CODE REVIEW

File: app/templates/components/ui-table/cell/cell-input.hbs [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<div class="inline field">
```
Comment (geekyd) [inline app/templates/components/ui-table/cell/cell-input.hbs]: Call the file `cell-input-number`

Comment (sumedh123) [inline app/templates/components/ui-table/cell/cell-input.hbs]: @geekyd Done.

**Comment by niranjan94** [comment]
Deployment link: https://pr-740-open-event-frontend-fossasia.surge.sh

**Comment by niranjan94** [comment]
Deployment link: https://pr-740-open-event-frontend-fossasia.surge.sh

**Comment by niranjan94** [comment]
Deployment link: https://pr-740-open-event-frontend-fossasia.surge.sh

**Pull request merged**